client/app/js/crypto/module.js

```diff
@@ -119,20 +122,26 @@ angular.module('GLBrowserCrypto', [])
 // containing form's ngModelController NOT the ngModel bound to the value of the 
 // text-area itself.
 .directive('pgpPubkeyValidator', function() {
-  
   // Checks to see if passed text is an ascii armored GPG public key.
   function validatePubKey(textInput) {
     // Check for obvious problems.
     if (typeof textInput !== 'string') {
       return false; 
     }
+
     var s = textInput.trim();
+
+    if (s === '') {
+      return true;
```

Comment:
yep i totally know.

the reason i did the following is that using this wizard allow empty keys and marking the form invalid caused pain :)

but i now got an idea that if solves the issue will make me remove this immediatly.

going to comment the code below and then pushing a small commit.